src/SFML/Window/DRM/InputImplUDev.cpp

```diff
@@ -87,8 +87,8 @@ bool systemDown()
 
 void uninitFileDescriptors()
 {
-    for (std::vector<int>::iterator itr = fileDescriptors.begin(); itr != fileDescriptors.end(); ++itr)
-        close(*itr);
+    for (const auto fileDescriptor : fileDescriptors)
```

Comment:
I used a value since this is an `int` and using a reference wouldn't be helpful. I'm fine using a reference though if you want.

Reply:
You're still referencing a specific value in a vector. I think you'll save yourself a lot of brain power, if we default to auto& instead of always thinking about whether we're now just reading or also writing data, and it keeps it consistent with everything else. (Note how all the other changes use auto&).